code/game/objects/items/weapons/storage/lunchbox.dm

```diff
@@ -496,6 +509,39 @@
 			desc = "A little plastic lunchbox. This one has a picture of Amy decorating the side, the nymphomaniac urban legend of Nanotrasen space stations."
 			icon_state = "lunchbox_pinup"
 			item_state = "toolbox_lightblue"
+		if(15)
+			name = "Ablative lunchbox"
+			desc = "A little plastic lunchbox. This one is has a decorative design similar to an ablative vest on the side."
+			icon_state = "lunchbox_ablative"
+			item_state = "toolbox_lightblue2"
+		if(16)
+			name = "Nuclear lunchbox"
+			desc = "A little plastic lunchbox. This one has a design on the side that looks similar to a nuclear fission device."
+			icon_state = "lunchbox_nuke"
+			item_state = "toolbox_brown"
+		if(17)
+			name = "ERT lunchbox"
+			desc = "A little plastic lunchbox. This one has a scene decorating the side that depicts four ERT members striking action poses."
+			icon_state = "lunchbox_ert"
+			item_state = "toolbox_lightblue"
+		if(18)
+			name = "Shuttle lunchbox"
+			desc = "A little plastic lunchbox. This one is decorated to look like a little escape shuttle."
+			icon_state = "lunchbox_shuttle"
+			item_state = "toolbox_lightblue2"
+
+/obj/item/weapon/storage/lunchbox/plastic/nt/random/bullet_act(var/obj/item/projectile/P) // Ablative lunchboxes protect lunch from lasers!
```

Comment:
this is fine for just one gimmick version, but if you ever give alternate features to more lunchboxes you need to make this oop by making them subtypes